.github/docker/Dockerfile.gorgone-testing-bookworm

```diff
@@ -0,0 +1,28 @@
+FROM debian:bookworm
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Set locale
+RUN apt-get update && \
+    apt-get install -y locales libcurl4-openssl-dev curl wget zstd jq lsb-release mariadb-client iproute2 && \
+    apt-get install -y ca-certificates apt-transport-https software-properties-common gnupg2 && \
+    localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
+
+ENV LANG=en_US.UTF-8
+
+# Install required packages and Robotframework
+RUN apt-get update && \
+    apt-get install -y \
+    python3 \
+    python3-dev \
+    python3-pip && \
+    pip3 install --break-system-packages --no-cache-dir \
+    robotframework \
+    robotframework-examples \
+    robotframework-databaselibrary \
+    pymysql \
+    robotframework-requests && \
+    echo "deb https://packages.centreon.com/apt-standard-24.05-testing/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/centreon.list && \
```

Comment:
should we make the "24.05" part of the url variable ? (since for develop it is always a moving version)